Spec/RealtimeClientConnection.swift

```diff
@@ -1461,13 +1448,11 @@ override class var defaultTestSuite : XCTestSuite {
                     expect(keys).to(haveCount(max))
                 }
 
-            }
-
             // RTN10
-            context("serial") {
+            
 
                 // RTN10a
-                it("should be -1 once connected") {
+                func test__042__Connection__serial__should_be__1_once_connected() {
```

Comment:
This is an unfortunate renaming, not sure what to do about it, maybe we just go and fix it up manually afterwards. I'll see how many other names have symbols that give meaning.

Reply:
I added functionality to the migrator to warn when it thinks it's encountered a test where removing a symbol might change the name. And I've opened https://github.com/ably/ably-cocoa/pull/1242 to address these warnings by changing the test descriptions.